### ISSUE TITLE
treatment of section/@removeInRFC not defined

### ISSUE THREAD

**Issue by reschke** [rfc-format/draft-iab-html-rfc-bis#6]
it is for the note element though

**Comment by hildjj** [comment]
Agree.  The intent here is that in I-D mode, the text stays in, but is styled to show that it will be removed.  In RFC mode, the preptool removes the text.
